### PR TITLE
AUDIO: Episode page fix midterm banner

### PR DESCRIPTION
**Pull request by ajwl**
## What does this change?

The Midterm banner was displaying the wrong colour text which was hard to read 

## Screenshots

BEFORE 
screen shot 2018-11-05 at 10 32 00

AFTER 
screen shot 2018-11-06 at 11 46 40

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
I'm surprised @Joshua-David did not ask you to add a padding

File: static/src/stylesheets/module/content-garnett/_media.head.scss [hunk 1]
```diff
@@ -52,3 +52,19 @@
         background: $brightness-20;
     }
 }
+
+.content--media.content--pillar-news {
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/content-garnett/_media.head.scss]: Is it the only pillar affected? (others are `opinion`, `lifestyle`, `arts`, and `sport`)

**Review by regiskuckaertz — CHANGES_REQUESTED** [review]
After discussing the matter with @akemitakagi we came to the following decisions:

1. the background should disappear (across all pillars)
2. the area of the header should expand vertically to include the whole kicker (right now it is bleeding into the content area)

**Comment by ajwl** [comment]
Okay now changed @regiskuckaertz

File: static/src/stylesheets/module/content-garnett/_media.head.scss [hunk 2]
```diff
@@ -52,3 +52,16 @@
...
     }
 }
+
+/* Styling fix for special topic label */
+.content--media {
+    header {
+        .content__labels.content__labels--immersive {
+            background: none;
+            .badge-slot {
+                margin-bottom: 16px;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/content-garnett/_media.head.scss]: Is this taking care of point 2?

Comment (ajwl) [reply static/src/stylesheets/module/content-garnett/_media.head.scss]: The banner has different behaviour at smaller breakpoints. But Akemi was happy with this implementation both above and below the change

Comment (ajwl) [reply static/src/stylesheets/module/content-garnett/_media.head.scss]: The text is intended to sit *below* the header rather than in it

**Review by regiskuckaertz — APPROVED** [review]
If @akemitakagi 's happy then so am I 🎉

**Comment by ajwl** [comment]
Thank you

**Pull request merged**